scripts/generateCliCode.ts

```diff
@@ -95,6 +95,14 @@ function capitalizeParts(s: string) {
   return s.split('.').map(capitalize).join('.');
 }
 
+function kebabCase(s: string) {
```

Comment:
This is only ever used as part of `kebabConvert`; it might be useful to make it local to that (so there's no confusion about which function to call).

Reply:
Sure. Also I'm calling it `kebabCase` to be consistent with `lodash.kebabCase`, with a comment on how it differs (otherwise I would have used it).

Also, there's no need for the inner `kebabConvertPart` because the matcher never looks at `.`s, and requires the capital letters to be preceded by a lower-case alpha.